### DISCUSSION TITLE
Character desync and teleportation issue

### DISCUSSION THREAD

**Discussion by Ksarper** [FakeFishGames/Barotrauma#14327] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

_I have searched the issue tracker to check if the issue has already been reported and I did find it, however, the conditions are different, at least in that I use mods and it doesn't seem to be an issue at all, based on the fact that there were no mods used in other issue report. And in my case, teleporting is much more aggressive and also aborts any activity, i.e. I lose control on sonar or the interface gets closed._

For some unknown reason, when the load on the server increases, the character starts teleporting to a random points of the map. This happens not only with the host, but also with clients (**Not simultaneously!**).

I've attached two videos, the first one demonstrates the problem in an active campaign. In the second video I created a new campaign with the same mods and on the same submarine and used teleport to change the starting location. In the second video you can see that teleports happened 1 or 2 times when encountering monsters and, as I think, which type of monsters is not important.

### Attached files:
[Active campaign demo.webm](https://github.com/user-attachments/assets/dcd5eab3-f90a-456c-b508-f05a14857785)
[New campaign demo.webm](https://github.com/user-attachments/assets/df5bc9da-478e-40b1-af97-ef2d8f58360d)
[My modlist.txt](https://github.com/user-attachments/files/16348511/My.modlist.txt)
[My steam workshop mod collection](https://steamcommunity.com/sharedfiles/filedetails/?id=3255981801)
[My save file.zip](https://github.com/user-attachments/files/16348566/My.save.file.zip)

I've tried to find a mod that can provoke this, but we have been playing with a certain mod build for a long time and have not changed it, the problem is relatively new for me.

### What I noticed based on the tests:
- Items cause a heavy load on the server, which can create desync?
- The type of monster doesn't matter.
- The problem only appears in open water sailing.
- This problem does not happen at the same time for host and client, I don't know if it can happen at the same time or if it's just a coincidence.

### Ad. info:
- I've heard that piezocrystals can cause problems, but we have a mod that fixes it.
- This problem has haunted us on other boats as well.

### My system:
- Windows 10 Pro x64
- Intel i5 12400F
- NVIDIA 1050Ti
- 100 mbit/s

P.S: sorry for the large post, I tried to provide as much information as possible, I hope I didn't make it difficult to understand.

### Reproduction steps

1. Start the server using Lua.
2. Continue the campaign.
3. Wait for an encounter with enemies (it must be in the open sea).

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/discussions/14118

**Reply by Ksarper** [reply]
that is a shame, can't play the game
